app/constants/errorMessages.json

```diff
@@ -1,5 +1,6 @@
 {
     "alreadyPrinting": "You seem to be printing already, please close your current print dialog before attempting to print again.",
     "missingGlobal": "Missing cryptographic parameters, please connect to a node to load these.",
+    "unableToReachWalletProxy": "Unable to reach the Wallet Proxy.",
```

Comment:
I guess we can't expect the user to know the wallet proxy, but what can we say instead?

Reply:
`An error occurred while trying to fetch new transactions.` or something like that.